### DISCUSSION TITLE
duplicate tab command does not duplicate the current directory location in the new tab

### DISCUSSION THREAD

**Discussion by duggalrohi** [JanDeDobbeleer/oh-my-posh#866] [Troubleshoot]
I am running terminal preview, when I try to duplicate tab from a location, it would go to default user directory instead of the current directory I am working with.

**Comment by JanDeDobbeleer** [comment]
For that to work you need to create a custom theme ([exporting your current config](https://ohmyposh.dev/docs/pwsh#override-the-theme-settings) will suffice if you have a standard theme) and set `"osc99": true`. The reason this is disabled by default is that it's not compatible with every terminal out there.
